pymongo/change_stream.py

```diff
@@ -197,23 +210,23 @@ def _resume(self):
             pass
         self._cursor = self._create_cursor()
 
-    def close(self):
+    def close(self) -> None:
         """Close this ChangeStream."""
         self._cursor.close()
 
-    def __iter__(self):
+    def __iter__(self) -> Iterator[Mapping[str, Any]]:
```

Comment:
Should this be "ChangeStream[_DocumentType]" instead?

Reply:
Yep